.github/workflows/release.yml

```diff
@@ -39,9 +41,56 @@ jobs:
           python -m twine upload --username "${TWINE_USER}" --password "${TWINE_PASSWORD}" --skip-existing --repository-url ${REPO_URL} dist/*.whl
         env:
           REPO_URL: "https://upload.pypi.org/legacy/"
-      - if: always()
-        uses: elastic/apm-pipeline-library/.github/actions/notify-build-status@current
+
+  publish-lambda-layers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build lambda layer zip
+        run: ./dev-utils/make-distribution.sh
+      - name: Publish lambda layers to AWS
+        run: |
+          # Convert v1.2.3 to ver-1-2-3
+          VERSION=${GITHUB_REF_NAME/v/ver-}
+          VERSION=${VERSION//./-}
+        
+          ELASTIC_LAYER_NAME="elastic-apm-python-${VERSION}" .ci/publish-aws.sh
+      - uses: actions/upload-artifact@v3
+        with:
+          name: arn-file
+          path: ".arn-file.md"
+          if-no-files-found: error
+
+  github-draft:
+    needs:
+      - publish-lambda-layers
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/download-artifact@v3
+        with:
+          name: arn-file
+      - run: >- 
+          gh release create "${GITHUB_REF_NAME}"
+          --title="${GITHUB_REF_NAME}"
+          --generate-notes
+          --notes-file=".arn-file.md"
+        env:
+          GH_TOKEN: ${{ github.token }}
```

Comment:
@basepi 

I'm creating a github release draft here, which can be adjusted and published manually later.

I did this so we have the arn table already in the draft.

<details>
<summary> example arn table created during testing </summary>

### Elastic APM Python agent layer ARNs

|Region|ARN|
|------|---|
|ap-northeast-1|arn:aws:lambda:ap-northeast-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|ap-northeast-2|arn:aws:lambda:ap-northeast-2:627286350134:layer:jcal-dev-elastic-apm-python:6|
|ap-northeast-3|arn:aws:lambda:ap-northeast-3:627286350134:layer:jcal-dev-elastic-apm-python:6|
|ap-south-1|arn:aws:lambda:ap-south-1:627286350134:layer:jcal-dev-elastic-apm-python:8|
|ap-southeast-1|arn:aws:lambda:ap-southeast-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|ap-southeast-2|arn:aws:lambda:ap-southeast-2:627286350134:layer:jcal-dev-elastic-apm-python:6|
|ca-central-1|arn:aws:lambda:ca-central-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|eu-central-1|arn:aws:lambda:eu-central-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|eu-north-1|arn:aws:lambda:eu-north-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|eu-west-1|arn:aws:lambda:eu-west-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|eu-west-2|arn:aws:lambda:eu-west-2:627286350134:layer:jcal-dev-elastic-apm-python:6|
|eu-west-3|arn:aws:lambda:eu-west-3:627286350134:layer:jcal-dev-elastic-apm-python:6|
|sa-east-1|arn:aws:lambda:sa-east-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|us-east-1|arn:aws:lambda:us-east-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|us-east-2|arn:aws:lambda:us-east-2:627286350134:layer:jcal-dev-elastic-apm-python:6|
|us-west-1|arn:aws:lambda:us-west-1:627286350134:layer:jcal-dev-elastic-apm-python:6|
|us-west-2|arn:aws:lambda:us-west-2:627286350134:layer:jcal-dev-elastic-apm-python:6|



</details>